### PR TITLE
Update connect-to-microsoft-365-powershell.md

### LINKED ISSUE #11781: Steps do not work as described
Under Step 2, I ran Connect-MsolService. PowerShell 7.3.0 returned "The term 'Connect-MsolService' is not recognized."

I had to run Import-Module MSOnline first in order to get Connect-MsolService to run at all.

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 55574af9-0105-dcd1-7eb0-77513df4e54f
* Version Independent ID: 5fe2849b-d39a-4d75-a5ce-e8f577c9a161
* Content: [Connect to Microsoft 365 with PowerShell - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/connect-to-microsoft-365-powershell?view=o365-worldwide#connect-with-the-azure-active-directory-powershell-for-graph-module)
* Content Source: [microsoft-365/enterprise/connect-to-microsoft-365-powershell.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/connect-to-microsoft-365-powershell.md)
* Service: **microsoft-365-enterprise**
* GitHub Login: @kelleyvice-msft
* Microsoft Alias: **kvice**

### PR DESCRIPTION
**Pull request by msbemba**
Updated steps to import module first

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11781

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/connect-to-microsoft-365-powershell.md [hunk 1]
```diff
@@ -140,12 +140,13 @@ These steps are required only one time on your computer. But you'll likely need
...
 1.  If you're not running Windows 10, install the 32-bit version of the Microsoft Online Services Sign-in Assistant: [Microsoft Online Services Sign-in Assistant for IT Professionals RTW](https://download.microsoft.com/download/7/1/E/71EF1D05-A42C-4A1F-8162-96494B5E615C/msoidcli_32bit.msi).
     
-2. Follow these steps to install the Microsoft Azure Active Directory Module for Windows PowerShell:
+2. Follow these steps to install and import the Microsoft Azure Active Directory Module for Windows PowerShell:
     
    1. Open an elevated Windows PowerShell command prompt (run Windows PowerShell as an administrator).
    1.  Run the **Install-Module MSOnline** command.
    1. If you're prompted to install the NuGet provider, type **Y** and press Enter.
    1. If you're prompted to install the module from PSGallery, type **Y** and press Enter.
+   1. Run the **Import-Module MSOnline** to import the module.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/connect-to-microsoft-365-powershell.md]:
```suggestion
   1. Run the **Import-Module MSOnline** command to import the module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kelleyvice-msft Gentle reminder.

**Pull request merged**